datadog_checks_dev/datadog_checks/dev/tooling/commands/ci/setup.py

```diff
@@ -0,0 +1,60 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import platform
+import subprocess
+
+import click
+
+from ...constants import get_root
+from ...testing import get_tox_envs
+from ..console import CONTEXT_SETTINGS, echo_debug, echo_info
+
+
+def display_action(script_file):
+    display_header = f'Running: {script_file}'
+    echo_info(f'\n{display_header}\n{"-" * len(display_header)}\n')
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Run CI setup scripts')
+@click.argument('checks', nargs=-1)
+@click.option('--changed', is_flag=True, help='Only target changed checks')
+def setup(checks, changed):
+    """
+    Run CI setup scripts
+    """
+    cur_platform = platform.system().lower()
+    scripts_path = os.path.join(get_root(), '.azure-pipelines', 'scripts')
+    echo_info("Run CI setup scripts")
+    if checks:
+        if checks[0] == 'skip':
+            echo_info('Skipping set up')
+        else:
+            echo_info(f'Checks chosen: {", ".join(checks)}')
+    else:
+        echo_info(f'Checks chosen: changed')
+
+    check_envs = get_tox_envs(checks, every=True, sort=True, changed_only=changed)
+
+    for check, _ in check_envs:
+        scripts_path = os.path.join(scripts_path, check)
+
+        if not os.path.isdir(scripts_path):
+            echo_debug(f"Skip! No scripts for check `{check}` at: `{scripts_path}`")
```

Comment:
This will never occur, no? https://github.com/DataDog/integrations-core/blob/f09fa0447d079ffacebe63af1fcb0d23c44bab08/datadog_checks_dev/datadog_checks/dev/tooling/commands/console.py#L70-L72

Reply:
I guess no in normal mode, but if we need to debug, we can just create a PR to trigger build with minimal change by adding `--debug`.